node/nodeTransactions_test.go

```diff
@@ -500,17 +502,42 @@ func TestNode_GetTransaction_ShouldNotFindAndReturnUnknown(t *testing.T) {
 	assert.Error(t, err)
 }
 
-func getCacherHandler(find bool) func() dataRetriever.ShardedDataCacherNotifier {
+func getCacherHandler(find bool, cacherType string) func() dataRetriever.ShardedDataCacherNotifier {
```

Comment:
You could have created separate functions to create the dummy pools (without the switch). But ok for now.